### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
That's a draft PR. It's my first serious PR, so I hope to get feedback on what I'm doing wrong. Currently my plan is to fix tests which fail locally.

There are locale-related failures of tests. I suggest making them independent from the locales.

### CODE REVIEW

File: src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs [hunk 1]
```diff
@@ -813,17 +813,11 @@ public async Task Pound_r_nuget_does_not_repeat_notifications_for_previous_r_nug
 
         [Theory]
         [InlineData(Language.CSharp)]
-        //[InlineData(Language.FSharp)]   /// Reenable when --- https://github.com/dotnet/fsharp/issues/8775
+        [InlineData(Language.FSharp)]
```
Comment (WhiteBlackGoose) [inline src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs:816]: Here the issue was closed, so I reenabled it. 🦆

File: src/Microsoft.DotNet.Interactive.PowerShell.Tests/PowerShellKernelTests.cs [hunk 2]
```diff
@@ -155,9 +155,7 @@ public async Task PowerShell_native_executable_output_is_collected()
         {
             var kernel = CreateKernel(Language.PowerShell);
 
-            var command = Platform.IsWindows
-                ? new SubmitCode("ping.exe -n 1 localhost")
-                : new SubmitCode("ping -c 1 localhost");
+            var command = new SubmitCode("dotnet --help");
```
Comment (WhiteBlackGoose) [inline src/Microsoft.DotNet.Interactive.PowerShell.Tests/PowerShellKernelTests.cs:158]: `ping` has literally nothing in common on my Russian Windows machine and English Linux machine. Here:

Mine: `обмен пакетами с desktop-1if9leb [::1] с 32 байтами данных:`
CI: `ping localhost (127.0.0.1) 56(84) bytes of data.\n`

So let it be something else

**Comment by WhiteBlackGoose** [comment]
I *think* I fixed all such tests that I could run and that failed.

**Comment by WhiteBlackGoose** [comment]
@jonsequitur it fails on test `server_listens_on_ephemeral_port`, which I didn't even change. The test expects an opened local port to be at least 10_000, whereas on CI it opened on 7645. Should we/I/you somehow fix it?

// also, when my PRs will be reviewed? :).

**Comment by WhiteBlackGoose** [comment]
@vzarytovskii isn't it better to fix the test rather than wait until it opens under a >10000 port? 😄

**Comment by vzarytovskii** [comment]
> @vzarytovskii isn't it better to fix the test rather than wait until it opens under a >10000 port? 😄 

I've restarted it not to fix the CI run, but to see whether it will reproduce or not, since on windows 10, the ephemeral port range should be in between 49152 and 65535.

It can be one of two things - either we hit the misconfigured agent, or default behaviour had changed.

Also question is whether we care if port is real "ephemeral" and has to fit in the "official" range, or we're fine with it being >= 1025.

**Comment by WhiteBlackGoose** [comment]
Now it passes ¯\_(ツ)_/¯

**Comment by WhiteBlackGoose** [comment]
Yes, @colombod.

**Review by colombod — APPROVED** [review]

**Pull request merged**